extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -228,6 +228,10 @@ const BLOCK_SRA_COMBINERS_ = [
       if (currEids) {
         currEids.split(',').forEach(eid => eids[eid] = 1);
       }
+      const deid = /(^|,)deid=(\d+)/i.exec(instance.win.location.hash);
```

Comment:
Maybe consider having this be an exported const in utils.js, to avoid risk of this getting out of sync with the previous usage.